packages/k8s/k0s/collection.yaml

```diff
@@ -0,0 +1,79 @@
+packages:
+  - name: k0s-openrc
+    category: k8s
+    version: "1.31.3"
+    k3s_version: "0"
+    labels:
+      github.owner: "k0sproject"
+      github.repo: "k0s"
+      autobump.sed_script: 's/\+k0s.[0-9]//g'
```

Comment:
This is similar to what we have in k3s, so I'm sure it will work, just keep in mind that by doing this we ignore patched versions e.g. v1.31.3+k0s.1 will not be considered because version v1.31.3+k0s.0 is already installed.

this is an internal problem that we need to solve. I thought I had a ticket for it but I cannot find it, will relate to this one instead in the meantime https://github.com/kairos-io/kairos/issues/1934

No need to worry on your side about this atm, just a not to self kind of thing